fireant/queries/builder/dataset_blender_query_builder.py

```diff
@@ -139,22 +138,16 @@ def _map_fields(fields):
             yield field.for_(field_map[field_from_blender])
 
     dataset_metrics = ordered_distinct_list_by_attr(_map_fields(metrics))
-    dataset_dimensions = _map_fields(dimensions)
-    dataset_filters = _map_fields(filters)
-    dataset_references = _map_fields(references)
+    dataset_dimensions = list(_map_fields(dimensions))
```

Comment:
Does this render the listify decorator above useless? (Not exactly sure what it does).

Reply:
You are right. I think I no longer noticed the listify decorator.